AnimatedWordCloud/Animator/TimelapseWordVector.py

```diff
@@ -37,7 +42,9 @@ def add(self, word: str, weight: float) -> None:
         :rtype: None
         """
 
-        heapq.heappush(self._word_heap, (weight, word))
+        # negate the weight to get the descending order
+        bisect.insort(self._word_bisect, (-weight, word))
```

Comment:
In my environment, I ran the following code:
container = [(-1, "1"),(-2, "2")]
bisect.insort(container, (-3, "3"))
print(container)

The result is "[(-3, '3'), (-1, '1'), (-2, '2')]"